core/templates/dev/head/domain/collection/SearchExplorationsBackendApiService.js

```diff
@@ -0,0 +1,49 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Service to search explorations metadata.
+ */
+
+oppia.factory('SearchExplorationsBackendApiService', [
+    '$http', 'alertsService', 'UrlInterpolationService', '$q',
+    function($http, alertsService, UrlInterpolationService, $q) {
+      var SEARCH_EXPLORATION_URL_TEMPLATE = (
```

Comment:
Pass in the url as a service, see `CollectionRightsBackendApiService:20` and define it in `core/templates/dev/head/pages/collection_editor/CollectionEditor.js`. That way it'll be easier to change the URL at a later date if need be.